src/components/buttons/button-table-edit.jsx

```diff
@@ -121,11 +122,7 @@ class ButtonTableEdit extends React.Component {
 	_handleChange = (inputName, event) => {
 		const state = {};
 
-		if (inputName === INPUT_NAMES.COLS || inputName === INPUT_NAMES.ROWS) {
-			state[inputName] = Math.min(event.target.value, MINIMUM_GRID_VALUE);
-		} else {
-			state[inputName] = event.target.value;
-		}
+		state[inputName] = event.target.value;
```

Comment:
Instead of:

```js
const state = {};

state[inputName] = event.target.value;

this.setState(state);
```

this could be:

```js
this.setState({
    [inputName]: event.target.value,
});
```

but funnily enough, your result is almost the same [as it used to be](https://github.com/liferay/alloy-editor/blob/a1532cac870dd25f58e4f8db89d2b4f7bac399d3/src/components/buttons/button-table-edit.jsx#L115-L118) before all the recent changes:

```js
const state = {};
state[inputName] = event.target.value;

this.setState(state);
```

Reply:
Added: https://github.com/liferay/alloy-editor/pull/1419#discussion_r469162192